### ISSUE TITLE
JSON within data returns numbers as strings

### ISSUE THREAD

**Issue by jrandall** [wtsi-hgi/treeserve#11]
e.g.: 

``` json
                      "*": "3.1296939190754969e-09",
                      "directory": "3.1296939190754969e-09",
                      "other": "3.1296939190754969e-09"
```

instead of:

``` json
                      "*": 3.1296939190754969e-09,
                      "directory": 3.1296939190754969e-09,
                      "other": 3.1296939190754969e-09
```

**Comment by ejames604** [comment]
Presumably this is to do with the json code you added ?
You need to make sure you are pulling things out of the Datums with the
right type. Maybe you need to add some casts ?

**Comment by jrandall** [comment]
Fixed in 2fc0325680e73db086fd6ef01c746f3f175db96c
